front/frontend.md

```diff
@@ -22,17 +22,17 @@
 - activity-navbar :
   > display description room and if a user wrinting in the input
 
-  ![html](../../logos/html.png) [html file](../../front/activity_bar.html)
+  ![html](../../logos/html.png) [html file](../front/src/activity_bar.html)
```

Comment:
Wrong path to the logo, so it's note displayed simply see markdown render at : https://github.com/mdl29/scratchy/blob/fd348da0ad69734e5407911513109d0f22119dfa/front/frontend.md

Reply:
Seems to be a github bug, hash isn't present in the logo path generated by the preview tool.